### DISCUSSION TITLE
Add documentation for `PR_BODY` input

### DISCUSSION THREAD

**Discussion by CalvinRodo** [BetaHuhn/repo-file-sync-action#175] [Ideas] [answered]
**🐞 Describe the bug**

The README.md does not have documentation for the PR_BODY property how it works and what the default is. 

Would also be nice to have an example for folks to follow.

**Comment by BetaHuhn** [comment, marked as answer]
Agreed, this was overlooked by me when the option was added with #80.

Will add more documentation when I have time in the next few days.

**Reply by BetaHuhn** [reply]
I added the `PR_BODY` to the list of [action inputs](https://github.com/BetaHuhn/repo-file-sync-action#%EF%B8%8F-action-inputs) and a as "[Add content to the PR body](https://github.com/BetaHuhn/repo-file-sync-action#add-content-to-the-pr-body)" example. 

@CalvinRodo let me know if you have any questions about the input.
